src/content/tutorials/build-a-rust-markdown-parser-webassembly/index.md

```diff
@@ -0,0 +1,156 @@
+---
+updated: 2020-09-12
+difficulty: Advanced
+---
+
+# Build a Markdown Parser with Rust and WebAssembly
+
```

Comment:
Let’s add a "Before you start" section here, like we do [in the other tutorials](https://github.com/cloudflare/workers-docs-engine/blob/19b0a89bd4b6cb37b8f965ed0213170c97945bad/src/content/tutorials/build-a-slackbot/index.md#L6-L10):

```markdown
import TutorialsBeforeYouStart from "../../_partials/_tutorials-before-you-start.md"

# Build a Markdown parser with Rust and WebAssembly

<TutorialsBeforeYouStart/>

<!-- ... -->
```

(Note: “parser” also lowercased.)